Update Package.swift

pruthvikar opened the pull request:
Update Package.swift to use Vapor 2 release dependencies

pruthvikar committed:
Update Package.swift

Update Package.swift to use Vapor 2 release dependencies

pruthvikar committed:
Update Package.swift

Pinning using major version

natebird commented on Package.swift:
```diff
@@ -9,9 +9,9 @@ let package = Package(
         .Package(url: "https://github.com/vapor-community/cpostgresql.git", beta),
```
Let's update cpostgresql line to `majorVersion: 2` as well. We can merge in this PR when the CPostgreSQL library is released as 2.0.
Remove the `beta` variable above as well. 

Thanks!

pruthvikar committed:
Update Package.swift

Updated cpostgresql to major version 2

vzsg committed:
Trigger CI

This is a simple cosmetic change to force Circle and Travis to run.
I tagged CPostgreSQL 2.0.0, so the package should now build.
